admin/acceptance.py

```diff
@@ -452,6 +453,51 @@ def eliot_output(message):
     sys.stdout.flush()
 
 
+def capture_journal(reactor, host, output_file):
+    """
+    SSH into given machine and capture relevant logs, writing them to
+    output file.
+
+    :param reactor: The reactor.
+    :param bytes host: Machine to SSH into.
+    :param file output_file: File to write to.
+    """
+    ran = run(reactor, [
+        "ssh",
+        b"-C",  # compress traffic
+        b"-q",  # suppress warnings
+        b"-l", 'root',
+        # We're ok with unknown hosts; we'll be switching away from
+        # SSH by the time Flocker is production-ready and security is
+        # a concern.
+        b"-o", b"StrictHostKeyChecking=no",
+        # The tests hang if ControlMaster is set, since OpenSSH won't
+        # ever close the connection to the test server.
+        b"-o", b"ControlMaster=no",
+        # Some systems (notably Ubuntu) enable GSSAPI authentication which
+        # involves a slow DNS operation before failing and moving on to a
+        # working mechanism.  The expectation is that key-based auth will
+        # be in use so just jump straight to that.  An alternate solution,
+        # explicitly disabling GSSAPI, has cross-version platform and
+        # cross-version difficulties (the options aren't always recognized
+        # and result in an immediate failure).  As mentioned above, we'll
+        # switch away from SSH soon.
+        b"-o", b"PreferredAuthentications=publickey",
```

Comment:
Not sure when we'll switch away from SSH for this (though it would certainly be nice to have a real log aggregation system).